### PR TITLE
test: (e2e) add datetime picker e2e tests core

### PR DESCRIPTION
**Pull request by nikvalor**
#### Please provide a link to the associated issue.
[datetime-picker](https://app.zenhub.com/workspaces/fundamental-library-5f32933d64202b0012e3572e/issues/sap/fundamental-ngx/4811)
#### Please provide a brief summary of this pull request.
Added datetime picker e2e tests core (+ visual regression)

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by nikvalor** [comment]
Project: null Platform B
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null Platform A 
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null CORE 
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null CORE 
 Url: null 
 This link will expire at

File: e2e/wdio/core/tests/date-time-picker.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+        expect(getText(selectYearButton)).toBe(year2030.toString());
+    });
+
+    it('verify by default today date is focused', () => {
+        const activeButtonsLength = getElementArrayLength(activeDateTimePickerButton);
+        for (let i = 1; i < activeButtonsLength; i++) {
+            sendKeys(['Escape']);
+            scrollIntoView(activeDateTimePickerButton, i);
+            click(activeDateTimePickerButton, i);
+            expect(isElementDisplayed(calendarExpanded)).toBe(true);
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/date-time-picker.e2e-spec.ts:112]: need to add 'i'

File: e2e/wdio/core/tests/date-time-picker.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+        click(cancelButton);
+        expect(getAttributeByName(activeDateTimePickerInput, 'ng-reflect-model'))
+            .not.toEqual(date);
+    });
+
+    it('verify disabled date time picker', () => {
+        const disabledButtonsArr = elementArray(disabledDateTimePickerButton);
+        for (let i = 0; i < disabledButtonsArr.length; i++) {
+            expect(isElementClickable(disabledDateTimePickerButton)).toBe(false);
+            expect(isElementClickable(disabledDateTimePickerInput)).toBe(false);
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/date-time-picker.e2e-spec.ts:166]: need to add 'i' to both of these

**Review by Seamoo13 — CHANGES_REQUESTED** [review]

File: e2e/wdio/core/tests/date-time-picker.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+        const buttonsLength = getElementArrayLength(datePickerButton);
+        const inputsLength = getElementArrayLength(datePickerInput);
+        expect(buttonsLength).toEqual(inputsLength);
+        for (let i = 1; i < buttonsLength; i++) {
+            expect(isElementDisplayed(datePickerButton, i)).toBe(true);
+            expect(isElementDisplayed(datePickerInput, i)).toBe(true);
+        }
+    });
+
+    it('verify on click on the date picker button', () => {
```
Comment (AntonOlkhovskyi) [inline e2e/wdio/core/tests/date-time-picker.e2e-spec.ts:84]: could you be more specific, what should happen en on that click ?

File: e2e/wdio/core/tests/date-time-picker.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,430 @@
...
+            click(activeDateTimePickerButton, i);
+            click(selectMonthButton);
+            waitForElDisplayed(filterCalendarValue('month'));
+            click(buttonFirstMonth);
+            waitForElDisplayed(filterCalendarValue('day'));
+            click(activeDateTimePickerButton, i);
+        }
+    });
+
+    it('verify simple datetime picker', () => {
```
Comment (AntonOlkhovskyi) [inline e2e/wdio/core/tests/date-time-picker.e2e-spec.ts:207]: ...has correct default date

**Review by AntonOlkhovskyi — CHANGES_REQUESTED** [review]

File: e2e/wdio/platform/tests/date-time-picker.e2e-spec.ts [hunk 5]
```diff
@@ -115,7 +115,7 @@ describe('Datetime picker suite', function() {
             .toContain(getCSSPropertyByName(dateTimePickerPage.dayInCalendarButtonByValue('1'), 'background-color').value);
     });
 
-    it('Verify When the user selects cancel the action is aborted and the input field remains unchanged.', () => {
+    fit('Verify When the user selects cancel the action is aborted and the input field remains unchanged.', () => {
```
[the remark this answers is not in the document]
Comment (Seamoo13) [reply e2e/wdio/platform/tests/date-time-picker.e2e-spec.ts:118]: please remove 'f'

**Review by Seamoo13 — CHANGES_REQUESTED** [review]

**Review by Seamoo13 — APPROVED** [review]

**Review by AntonOlkhovskyi — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**